sale_financial_risk_pos_compatibility/models/pos_session.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models
+
+
+class PosSession(models.Model):
+    _inherit = "pos.session"
+
+    def _get_pos_ui_res_users(self, params):
+        user_dict = super(PosSession, self)._get_pos_ui_res_users(params)
+        user_id = user_dict.get("id")
+        if user_id and (user := self.env["res.users"].browse(user_id)).exists():
+            user_dict.update(
+                has_role_risk_manager=user.has_group(
+                    "account_financial_risk.group_overpass_partner_risk_exception"
+                )
+            )
+        return user_dict
```

Comment:
user will exist because the id is from the current user

```suggestion
        data = super()._get_pos_ui_res_users(params)
        user = self.env["res.users"].browse(data["id"])
        data["has_role_risk_manager"] = user.has_group(
            "account_financial_risk.group_overpass_partner_risk_exception",
        )
        return data
```

Reply:
Resolved